VagrantProvision.sh

```diff
@@ -552,11 +552,8 @@ rm -rf $HOME/tmp
 
 cd $HOOT_HOME
 
-rm -rf $HOOT_HOME/ingest
-mkdir -p $HOOT_HOME/ingest/processed
-
-rm -rf $HOOT_HOME/upload
-mkdir -p $HOOT_HOME/upload
+rm -rf $HOOT_HOME/userfiles/ingest/processed
+mkdir -p $HOOT_HOME/userfiles/ingest/processed
 
```

Comment:
Doesn't this get created by hoot-services servlet context now (so it could be removed entirely)?